app/mailers/subscription_mailer.rb

```diff
@@ -29,8 +29,11 @@ def notify_tag_added(node, tag, current_user)
     @node = node
     @current_user = current_user
     given_tags = node.tags.reject { |t| t == tag}
-    users_to_email = tag.followers_who_dont_follow_tags(given_tags)
-    users_to_email.each do |user|
+    users_to_email = tag.followers_who_dont_follow_tags(given_tags).pluck(:uid)
```

Comment:
Is this failing because you can't pluck if it returns empty? Perhaps we have to account for that possibility in this solution.